core/src/main/java/org/openstack4j/api/OSClient.java

```diff
@@ -354,5 +355,12 @@
      * @return the DNS service
      */
     DNSService dns();
+    
+    /**
+     * Returns the Server Event Service API
+     *
+     * @return the Server Event Service API
+     */
+    EventServices event();
```

Comment:
Provided this is scoped to compute, it is better to have this only in `ComputeService`. So it will be `os.compute()...`